### PR TITLE
fixes ZEN-21880.

### PR DESCRIPTION
**Pull request by itshane**
Account ID field has been added to 'Add EC2 Account' dialog.

### CODE REVIEW

File: ZenPacks/zenoss/AWS/EC2Account.py [hunk 1]
```diff
@@ -31,13 +31,15 @@ class EC2Account(Device):
...
     ec2secretkey = None
     linuxDeviceClass = None
     windowsDeviceClass = None
     _setDiscoverGuests = None
 
     _properties = Device._properties + (
+        {'id': 'ec2accountid', 'type': 'string'},
         {'id': 'ec2accesskey', 'type': 'string'},
         {'id': 'ec2secretkey', 'type': 'string'},
         {'id': 'linuxDeviceClass', 'type': 'string'},
```
Comment (pcarinhas) [inline ZenPacks/zenoss/AWS/EC2Account.py]: Although not part of your changes, these could cause errors:

EC2Account.py|77 col 20 error| indentation is not a multiple of four 
EC2Account.py|98 col 20 error| indentation is not a multiple of four

File: ZenPacks/zenoss/AWS/deviceloaders.py [hunk 2]
```diff
@@ -37,4 +37,4 @@ def load_device(
             collector='localhost'):
 
         return getFacade('aws', dmd).add_ec2account(
-            accountname, accesskey, secretkey, collector)
+            accountid, accountname, accesskey, secretkey, collector)
```
Comment (pcarinhas) [inline ZenPacks/zenoss/AWS/deviceloaders.py]: deviceloaders.py|40 col 13 error| undefined name 'accountid'

Comment (itshane) [inline ZenPacks/zenoss/AWS/deviceloaders.py]: Fixed

File: ZenPacks/zenoss/AWS/EC2Account.py (hunk 1 above)
Comment (itshane) [inline ZenPacks/zenoss/AWS/EC2Account.py]: Fixed

**Pull request merged**